xcdat/axis.py

```diff
@@ -191,9 +268,8 @@ def _align_lon_to_360(dataset: xr.Dataset, p_meridian_index: np.ndarray) -> xr.D
         The Dataset.
     """
     ds = dataset.copy()
-    lon: xr.DataArray = dataset.bounds._get_coords("lon").copy()
-    lon_name = lon.name
-    lon_bounds: xr.DataArray = dataset.bounds.get_bounds("lon").copy()
+    lon: xr.DataArray = get_axis_coord(ds, "X")
+    lon_bounds: xr.DataArray = dataset.bounds.get_bounds("X")
```

Comment:
Just making sure it is okay to drop the `.copy()` here.

Reply:
I dropped `.copy()` in this function and meant to add `.copy()` to `get_bounds()`. Thanks for the reminder.

https://github.com/xCDAT/xcdat/blob/51627fd4f9b5caaa2a0d04e6d464b41fa7077446/xcdat/bounds.py#L185-L193

When `.copy()` is and isn't needed:
```
# Coordinate variable
# .copy() not needed, this creates a deep copy
lat = ds.lat
lat.data[0] = None

# Data variable
# .copy() needed, otherwise a shallow copy is created (ref by memory)
lat_bnds = ds.lat_bnds.copy()
lat_bnds.data[0] = None
```